src/domain/entities/CustomText.ts

```diff
@@ -0,0 +1,30 @@
+import i18n from "../../locales";
+import { getKeys } from "../../types/utils";
+import { TranslatableText } from "./TranslatableText";
+
+export interface CustomText {
+    root_title: TranslatableText;
```

Comment:
Just naming conventions... I rarely saw `snake_case` being used in our projects. We usually use `camelCase` even if it is going to end up in datastore as json, and for the key below, we might use `kebab-case` for example. https://github.com/EyeSeeTea/training-app/blob/feature/customize-logo-greeting/src/domain/entities/TrainingModule.ts